target/linux/mvebu/cortexa9/base-files/etc/board.d/02_network

```diff
@@ -94,6 +95,10 @@ mvebu_setup_macs()
 		lan_mac=$label_mac
 		wan_mac=$label_mac
 		;;
+	wd,cloud-mirror-gen2)
+		label_mac=$(macaddr_canonicalize $(strings /dev/mtd7|grep -E '([0-9A-F]{2}[:])'))
```

Comment:
mac address is on ubi "config" or ubi "reserve2" in text file . ubi "reserve2" /dev/mtd7 is twice small and only contains basic OEM factory info:
```
 -rw-r--r--    1 root     root             1 Jan 10  2017 hidden_eccryption
-rw-r--r--    1 root     root            18 Jan  9  2017 mac_addr
-rw-r--r--    1 root     root             0 Jan  9  2017 mfg
-rw-r--r--    1 root     root             2 Jan  9  2017 model_id
-rw-r--r--    1 root     root            12 Jan 10  2017 wd_serial.txt
```

Reply:
Please add this as a comment into the code. Do not add the ls output.

Reply:
comment added